daprdocs/content/en/developing-applications/middleware/supported-middleware/middleware-uppercase.md

```diff
@@ -10,7 +10,7 @@ The uppercase [HTTP middleware]({{< ref middleware-concept.md >}}) converts the
 
 ## Component format
 
-In the following definition, the maximum requests per second are set to 10:
+In the following definition, it make content of request body into uppercase:
```

Comment:
This is already covered in #1318

Reply:
Came from another commit when I rebased from v1.0 to v1.1 so I leaving this unchanged.